homeassistant/components/bbox/sensor.py

```diff
@@ -27,6 +27,9 @@
 from homeassistant.util import Throttle
 from homeassistant.util.dt import utcnow
 
+import urllib3
+requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS = 'ALL:@SECLEVEL=1'
```

Comment:
To add to the above comment, please note, that this will change the default ciphers for ALL integrations (as in the whole of Home Assistant). Instead, it should be used in a way that others aren't affected by this change.

Reply:
Thanks for the information, I'm beginning but I'll to correct and look for the corresponding code.

I bow out to you @frenck  for your work on H.A

Thank you.